tests/Agent/UnitTests/Core.UnitTest/Configuration/DefaultConfigurationTests.cs

```diff
@@ -4439,8 +4439,93 @@ public void AzureFunctionServiceName_ShouldReturnServiceName_WhenEnvironmentVari
             Assert.That(result, Is.EqualTo("some-service-name"));
         }
 
+        #endregion Cloud
```

Comment:
Is this where the `Cloud` region should end?

Reply:
Well that's weird. Not sure how I swapped the begin and end. Good catch!